doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -86,6 +86,7 @@ Setting properties of a function's arguments
          the parameter name used in the function definition).  Unless `rename` is specified,
          the list of :n:`@name`\s must be a prefix of the formal parameters, including all implicit
          arguments.  `_` can be used to skip over a formal parameter.
+         This declares the name as non-implicit if either `clear implicits` is declared or at least one other name is declared implicit in the same list of :n:`@name`\s.
```

Comment:
```suggestion
         This declares :n:`@name` as non-implicit if `clear implicits` is specified or at least one other name is declared implicit in the same list of :n:`@name`\s.
```

A little unclear what "This" refers to.  If it's the previous sentence, maybe splice the sentences:  `... to skip over a formal parameter, which declares ...`  If it's the construct, maybe say `This construct declares the name ...`

Reply:
I made it more clear: https://github.com/coq/coq/pull/13293/commits/87be983b7623126320dc0b43303a48ea41b86f5c#